src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -77,11 +78,17 @@ public function doRun(InputInterface $input, OutputInterface $output)
      */
     protected function doRunCommand(Command $command, InputInterface $input, OutputInterface $output)
     {
+        $parentRun = parent::doRunCommand($command, $input, $output);
```

Comment:
this actually runs the command thus writes to the output no matter which command is ran, not good, should be called at the right place